src/host/ft_host/CanaryTests.cpp

```diff
@@ -67,7 +65,7 @@ void CanaryTests::LaunchV1Console()
 
     wil::unique_process_information ProcessInformation;
     VERIFY_WIN32_BOOL_SUCCEEDED(CreateProcessW(NULL,
-                                               CmdLineMutable.get(),
+                                               const_cast<LPWSTR>(CmdLine.c_str()),

```

Comment:
So this one should definitely use `.data()` -- CreateProcessW _requires_ a mutable string (!) and since `data` is the mutable variety of `c_str`, it's a better fit

Reply:
A mutable string, huh. That's not what I'd expected. You learn something everyday.